symplyphysics/laws/relativistic/relativistic_mass.py

```diff
@@ -0,0 +1,35 @@
+from sympy import Eq, solve, sqrt
+from sympy.physics.units import speed_of_light as c
```

Comment:
Don't rename to `c`

Reply:
exact same thing was done in other file and it was ok (energy_is_mass.py)

Reply:
Thanks. I will fix that file as well.